wandb/plot/confusion_matrix.py

```diff
@@ -4,13 +4,14 @@
 chart_limit = wandb.Table.MAX_ROWS
 
 
-def confusion_matrix(preds=None, y_true=None, class_names=None):
+def confusion_matrix(probs=None, y_true=None, preds=None, class_names=None):
```

Comment:
I appreciate having the "probs" first, but I wanted to note that this will break backwards compatibility with any earlier code using confusion_matrix(preds, y_true, class_names?)...I personally appreciate not needing to name all the arguments and often rely on their order alone. I also understand if we want to just require names for all the args as best practice and can document this going forward.

Reply:
Added a check for the length of the probs. I think that should do it.